app/schema/api/v4/models.rb

```diff
@@ -89,6 +89,21 @@ def lookup_patient
       )
     end
 
+    def patient_attribute
+      {type: :object,
+       properties: {
+         id: {"$ref" => "#/definitions/uuid"},
+         user_id: {"$ref" => "#/definitions/uuid"},
+         patient_id: {"$ref" => "#/definitions/uuid"},
+         height: {"$ref" => "#/definitions/non_empty_string"},
+         weight: {"$ref" => "#/definitions/non_empty_string"},
```

Comment:
thinking out loud - should we also be including the unit for height and weight. Different countries might follow different units for measuring those, for instance pounds/kilograms for weight.